README.md

```diff
@@ -24,6 +24,49 @@ $ npm install osseus-mongo
 }
 ```
 
+#### Simple usage
+
+* Configuration file / secret: `OSSEUS_MONGO_URI: 'mongodb://<mongosever>:<port>/<databasename>'`
+* Model file (example: models/User.js): 
+```javascript
+module.exports = (osseus) => {
+    const db = osseus.mongo
+    const mongoose = osseus.mongo.mongoose
+    const Schema = mongoose.Schema
+
+    const userSchema = new Schema({
+        name: String
+    });
+    const User = db.model('User', userSchema)
+    
+    osseus.mongo.myModels = osseus.mongo.myModels || {}
+    osseus.mongo.myModels.User = User
+}
+```
+* Require model after osseus init: `require('./models/User')(osseus);`
```

Comment:
the usage example should be generic, something like:
"access to the models can then be achieved by osseus.mongo.myModels["modelName"] from anywhere in the application"

const newUser = new osseus.mongo.myModels.User({
                name: 'Someone'
            })
             newUser.save()
                .then(() => {
                    console.log('User successfully saved')
                })
                .catch(err => {
                    throw err
                })